assets/js/googlesitekit/notifications/datastore/notifications.test.js

```diff
@@ -843,6 +843,224 @@ describe( 'core/notifications Notifications', () => {
 				expect( fetchMock ).toHaveFetchedTimes( 1 );
 			} );
 		} );
+
+		describe( 'pinNotification', () => {
+			it( 'should require a notification ID to be provided', () => {
+				const { pinNotification } =
+					registry.dispatch( CORE_NOTIFICATIONS );
+
+				expect( () => pinNotification() ).toThrow(
```

Comment:
Super trivial, the anonymous function isn't needed here:

```suggestion
				expect( pinNotification ).toThrow(
```